docs/DeveloperGuide.md

```diff
@@ -293,20 +293,23 @@ The following sequence diagram shows how deleting a session works:
 ![DeleteSessionSequenceDiagram](images/shion/DeleteSessionSequenceDiagram.png)
 
 It shares the same design considerations as what is mentioned in Add Session Feature.
+Note that: this feature is shared below, in deletion of an entire recurring session.
```

Comment:
Perhaps you want to use the note block that we have used for other sections.
![Uploading image.png…]()

Reply:
yep!